tests/core/tester/test_generate_vm_configuration.py

```diff
@@ -39,37 +55,53 @@ class Forks(enum.Enum):
             ),
         ),
         (
-            dict(homestead_start_block=1, spurious_dragon_block=3),
+            ((0, 'frontier'), (1, 'homestead'), (3, 'spurious-dragon')),
+            {},
             (
                 (0, Forks.Frontier),
                 (1, Forks.Homestead),
                 (3, Forks.SpuriousDragon),
             ),
         ),
         (
-            dict(tangerine_whistle_start_block=1),
+            ((0, 'homestead'), (1, 'tangerine-whistle')),
+            {},
             ((0, Forks.Homestead), (1, Forks.TangerineWhistle)),
         ),
         (
-            dict(homestead_start_block=1),
+            ((0, 'frontier'), (1, 'homestead')),
+            {},
             ((0, Forks.Frontier), (1, Forks.Homestead)),
         ),
         (
-            dict(homestead_start_block=1, dao_start_block=2),
+            ((0, 'frontier'), (1, 'homestead')),
+            {'dao_start_block': 2},
             ((0, Forks.Frontier), (1, Forks.Homestead)),
         ),
         (
-            dict(homestead_start_block=1, dao_start_block=False),
+            ((0, 'frontier'), (1, 'homestead')),
+            {'dao_start_block': False},
```

Comment:
I had a hard time guessing what `False` and `None` did... but I see that you added docs, and I don't have a better suggestion for the API, so I guess nothing to do here.